test/cli/premium_test.py

```diff
@@ -26,48 +28,43 @@ def __copy_cppcheck_premium(tmpdir):
                 {
                     "addons": [],
                     "productName": "NAME",
-                    "about": "NAME",
-                    "safety": true
+                    "about": "NAME"
                 }
                 """.replace('NAME', __PRODUCT_NAME))
 
     return exe
 
 
 def test_misra_c_builtin_style_checks(tmpdir):
-    # FIXME this test does not work in ci-windows.yml (release build)
-    if sys.platform == 'win32':
-        return
-
     test_file = os.path.join(tmpdir, 'test.cpp')
     with open(test_file, 'wt') as f:
         f.write('void foo() { int x; y = 0; }')
 
     exe = __copy_cppcheck_premium(tmpdir)
 
-    exitcode, _, stderr = cppcheck(['--premium=autosar', '--xml', test_file], cppcheck_exe=exe)
+    exitcode, _, stderr = cppcheck(['--premium=autosar', '--premium=safety', '--xml', test_file], cppcheck_exe=exe)
     assert exitcode == 0
     assert 'id="unusedVariable"' in stderr
     assert 'id="checkersReport"' in stderr
 
-    exitcode, _, stderr = cppcheck(['--premium=autosar', '--premium=safety-off', '--xml', test_file], cppcheck_exe=exe)
+    exitcode, _, stderr = cppcheck(['--premium=autosar', '--xml', test_file], cppcheck_exe=exe)
     assert exitcode == 0
     assert 'id="unusedVariable"' in stderr
     assert 'id="checkersReport"' not in stderr
 
-    exitcode, _, stderr = cppcheck(['--xml-version=3', test_file], cppcheck_exe=exe)
+    exitcode, _, stderr = cppcheck(['--xml-version=3', '--premium=safety', test_file], cppcheck_exe=exe)
     assert exitcode == 0
     assert '<safety/>' in stderr
 
-    exitcode, _, stderr = cppcheck(['--xml-version=3', '--premium=safety-off', test_file], cppcheck_exe=exe)
+    exitcode, _, stderr = cppcheck(['--xml-version=3', test_file], cppcheck_exe=exe)
     assert exitcode == 0
     assert '<safety/>' not in stderr
```

Comment:
Cppcheck Premium should enable safety by default. So I expect `<safety/>` in xml output unless it's explicitly turned off with `--premium=safety-off`.

Reply:
> Cppcheck Premium should enable safety by default.

That means that it is explicitly enabled in `cppcheck.cfg`, right? Should it even be allowed to be disabled by default if it is premium (i.e. `<safety>` is missing or set to `false`)?

What about `--no-safety`? May it also override the premium safety (default)? Or should that be limited to `--premium=safety-off`?

I would be fine having special handling for the safety flag. I just would have it properly defined.

Reply:
@danmar I need feedback on this to proceed.

Reply:
@danmar Still need feedback to proceed. 

FYI If this weren't premium-related I would have just defined things and merged it by now.